### ISSUE TITLE
generate a migration with no sql (so i can add the sql myself)

### ISSUE THREAD

**Issue by carchrae** [prisma/prisma#6206]
## Problem

tldr; i'd like `prisma migrate` to generate an empty migration where i can add the sql statements i wish to have in that migration.

longer: i'd like to use prisma to handle migrations, however, my schema and database are not perfectly mapped.  this leads to prisma generated migrations containing a lot of dangerous sql that i don't want to use (mainly dropping things).  what i do instead is rip out most of the sql and then either fill in my own migrations or remove the ones prisma generated which i don't want to keep.

## Suggested solution

i'd like a simple command `prisma migrate new` which just produces a blank migration with today's timestamp.

## Alternatives

* i can manually create a migration directory with the timestamp and create a `migration.sql` file.
* run `prisma migrate dev` and then replace/edit the  `migration.sql` file

## Additional context

this is by no means a big deal or show stopper, but this seems like a pretty easy feature to add in the 'nice to have' category.  until prisma schema and database are perfectly in sync, hand written migrations will likely be necessary in many contexts.  saving me some boilerplate copy paste would be nice.

**Comment by Jolg42** [comment]
Hi @carchrae 

It's actually already possible with `prisma migrate dev --create-only` if no change is detected an empty migration file will still be created and you will be able to edit it, put any SQL you like then run `prisma migrate dev` to execute it locally to verify it is valid.

After applying once you can still edit it and rerun `prisma migrate dev` as many times as needed, if Migrate detects that the computed checksum of the file changed it will try to rerun it.

**Comment by carchrae** [comment]
thanks for the quick reply @Jolg42 

the problem i have is that `prisma migrate dev --create-only` and `prisma migrate dev` will always think there are changes.  for example, i have had to comment out relations due to issues like this https://github.com/prisma/prisma/issues/5289 

i know in the perfect world, my schema would be mapped correctly, but when taking an existing database with all its lovely warts, i would still like to harness most of the migrate functionality (and the rest of prisma) but disable prisma generating the actual migration sql for me, since the schema isn't mapped perfectly.   

i know i can still use it and delete the generated sql, hence this isn't a massive problem for me.

i'll update the issue above to make that more clear.

**Comment by carchrae** [comment]
just to clarify, re the title change - i don't even want prisma to check for changes and start guessing about what needs to be changed!  i just want to generate a blank migration and ideally without wiping my dev database.

much like how rails does it (in the very basic case with an empty template): https://guides.rubyonrails.org/active_record_migrations.html#creating-a-migration

**Comment by tylerexpa** [comment]
any progress on this?  There are many reasons to want to apply a manually written migration file including on database features that prisma does not support.

**Comment by harrisonhoward** [comment]
This would be great
